utils/stripMarkdownLinks.test.ts

```diff
@@ -0,0 +1,13 @@
+import { stripMarkdownLinks } from './getTocContent'
+
+test('should return original string since there is no link', () => {
```

Comment:
```suggestion
test('should do nothing when there is no link markup', () => {
```

briefly describing the _case_ being tested will make it easier to understand test failures in the console. I'll update these.